src/PKSim.Infrastructure/ORM/DAS/DASDataRow.cs

```diff
@@ -261,60 +261,6 @@ public void SelectFromDB()
          if (RowState != DataRowState.Detached && RowState != DataRowState.Added)
             AcceptChanges();
       }
-
-      /// <summary>
```

Comment:
Most of the DAS Stuff is not required. SLowly but surely removing what we don't need. Goal is to not use this abstraction anymore